docs/synthetics/metrics.md

```diff
@@ -58,4 +58,85 @@ Browser monitor collects the below metrics for every resource loaded during the
 
 ### Custom Metrics
 
-You can script the Browser monitor to collect custom metrics that are specific to your website or use case. For example, you might need to measure the time it takes to display auto-suggestions on your website or collect & monitor the value from an element on your webpage.  To collect a custom metric use `context.setMetric(name, value)` method in the Browser monitor script. For more info on how to use this API, look into the examples while creating the Browser monitor. The metric name should be unique for this Synthetics App and will be automatically pre-pended with `synthetics.` prefix. You could use [Chart Builder](../dashboards/chart-builder/) to create a custom chart to monitor this metric. You could also create a threshold or anomaly alert on this metric.
\ No newline at end of file
+You can script the Browser monitor to collect custom metrics that are specific to your website or use case. For example, you might need to measure the time it takes to display auto-suggestions on your website or collect & monitor the value from an element on your webpage. You can use the `context.setMetric(name, value)` method in the Browser monitor script in order to define a custom metric. All metrics defined in monitors within one Synthetics App should have unique names and will be automatically pre-pended with a `synthetics.browser.custom.` prefix.
+
+Here's an example of a Browser monitor script which defines two custom metrics: the used JavaScript heap size and how long the combined duration of JavaScript execution is. For your convenience, this example is also listed as **Using Custom Metrics** under the **Browse Examples** section while creating the Browser monitor.
```

Comment:
Is "Browser monitor script" referring to UJS?

Reply:
Yeah, want me to change it to UJS?

Reply:
We should use consistent terminology in the docs and that terminology needs to match the terminology in the UI.